### DISCUSSION TITLE
v3.0.100

### DISCUSSION THREAD

**Discussion by 2881099** [dotnetcore/FreeSql#967] [General]
- 增加 南大通用 Gbase 国产数据库支持；
- 增加 ClickHouse 数据库语法支持；
- 增加 DbContext/Repository 比较变化方法 CompareState；
- 增加 DynamicFilter Custom 自定义解析；
- 增加 ToDataTableByPropertyName 动态查询功能；
- 优化 兼容排序 OrderBy(a => new {}) 语法；
- 优化 pgsql jsonb 映射，支持 List，mysql limit in 子查询；
- 优化 InsertOrUpdate<> 使用 InsertOrUpdate<list<>>时，提示友好异常。
- 修复 BulkCopy 与线程事务未传播的 bug；#962
- 修复 AsTreeCte + RereadSql 不能同时使用的 bug；#964
- 修复 FreeSql.Generator 工具生成model失败  [#882](https://github.com/dotnetcore/FreeSql/issues/882) 

<hr /><em>This discussion was created from the release <a href='https://github.com/dotnetcore/FreeSql/releases/tag/v3.0.100'>v3.0.100</a>.</em>

**Comment by luoyunchong** [comment]
## [.NET 链接数据库：证书链是由不受信任的颁发机构颁发的](https://www.cnblogs.com/dawenyang/p/12985957.html)

配置项增加
```
Encrypt=True;TrustServerCertificate=True;
```
